src/core/interrupts.c

```diff
@@ -40,7 +41,9 @@ inline void interrupts_init()
     interrupts_arch_init();
 
     if (cpu()->id == CPU_MASTER) {
-        interrupts_reserve(IPI_CPU_MSG, cpu_msg_handler);
+        if(!interrupts_reserve(IPI_CPU_MSG, cpu_msg_handler)) {
+            ERROR("Failed to assign IPI_CPU_MSG interrupt");
```

Comment:
```suggestion
            ERROR("Failed to reserve IPI_CPU_MSG interrupt");
```

Reply:
Thank you, fixed in https://github.com/bao-project/bao-hypervisor/pull/97/commits/878ad2b2ed85b8ed383a63e32650ee9363c624d8.